package.json

```diff
@@ -150,7 +152,7 @@
     "css-minimizer-webpack-plugin": "^4.0.0",
     "detect-port": "^1.3.0",
     "electron": "^20.0.2",
-    "electron-builder": "^23.3.3",
+    "electron-builder": "^23.6.0",
     "electron-devtools-installer": "^3.2.0",
```

Comment:
Was there a reason for changing the package.json? I don't think this was intentional so you can probably just remove the changes to package.json and package-lock.json.